core/templates/pages/exploration-editor-page/services/user-exploration-permissions.service.ts

```diff
@@ -42,6 +42,13 @@ export class UserExplorationPermissionsService {
     }
     return UserExplorationPermissionsService.permissionsPromise;
   }
+
+  async fetchPermissionsAsync(): Promise<ExplorationPermissions> {
```

Comment:
Can we use this function for getPermissionAsync function? [optional]

Reply:
Would be a good idea to make all backend calls from one function. Would make the code look cleaner.

DONE